.github/workflows/ci-model-regression-on-schedule.yml

```diff
@@ -0,0 +1,186 @@
+name: CI - Model Regression 
+
+on:
+  schedule:
+    - cron:  '1 23 * * */7'
+
+env:
+  GKE_ZONE: us-central1-a
+
+jobs:
+  read_test_configuration:
+    name: Reads tests configuration
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+    - name: Checkout master
+      uses: actions/checkout@v2
+
+    - name: Download yq
+      run: |
+        curl --location https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o yq
+        chmod +x yq
+
+    - run: test -f .github/configs/mr-test.yaml
+
+    - id: set-matrix
+      shell: bash
+      run: echo "::set-output name=matrix::$(./yq -j r .github/configs/mr-test-schedule.yaml)"
+
+  deploy_runner_gpu:
+    name: Deploy Github Runner - GPU
+    needs: read_test_configuration
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Download gomplate
+        run: |-
+          curl -o gomplate -sSL https://github.com/hairyhenderson/gomplate/releases/download/v3.6.0/gomplate_linux-amd64
+          chmod 755 gomplate
+
+      - name: Render deployment template
+        run: |-
+          ./gomplate -f .github/runner/deployment.yaml.tmpl -o runner_deployment.yaml
```

Comment:
I think it would help in the future to create some docs around this. how about adding a README in `.github/templates` describing how to install the tool locally and try out the templates?

Reply:
A good idea, I will do that.